packages/rfw/test/utils.dart

```diff
@@ -21,3 +22,15 @@ bool get isMainChannel {
 
 // See Contributing section of README.md file.
 final bool runGoldens = !kIsWeb && Platform.isLinux && isMainChannel;
+
+// slide to value for material slider in tests
+extension SlideTo on WidgetTester {
```

Comment:
can you make this a function instead of an extension? that would make it easier to find when reading the code (otherwise it looks like you have to look on WidgetTester).

See also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-extension